_templates/community.html

```diff
@@ -57,7 +57,7 @@ <h2 id="irc"><img src="/img/icons/ico_help.svg" class="titleicon" alt="Icon">{%
 </div>
 </div>
 
-<h2 id="nonprofit"><a name="{% translate non-profit anchor.community %}">{% translate nonprofit %}</a></h2>
+<h2 id="nonprofit-organizations"><a name="{% translate non-profit anchor.community %}">{% translate nonprofit %}</a></h2>
```

Comment:
I was indeed wondering why this didn't raise any error :) That all sounds good to me! I should have looked more carefully.